app/controllers/supplejack_api/harvester/sources_controller.rb

```diff
@@ -29,6 +32,7 @@ def show
 
       def update
         @source = Source.find(params[:id])
+        source_params['status_updated_at'] = Time.zone.now.to_datetime if source_params['status']
```

Comment:
no way we can put these params in to the request?

Reply:
I thought about that, but then we have to consider timezone of the browser etc.. by setting this on the server side we can guarantee it's the correct time.